agr_java_core/src/main/java/org/alliancegenome/neo4j/entity/PrimaryAnnotatedEntity.java

```diff
@@ -48,7 +47,7 @@ public class PrimaryAnnotatedEntity extends ConditionAnnotation implements Compa
 	@JsonView({View.PrimaryAnnotation.class, View.API.class})
 	protected String url;
 	@JsonView({View.PrimaryAnnotation.class, View.API.class})
-	protected GeneticEntity.CrossReferenceType type;
```

Comment:
@cmpich when this type gets serialized needs to match the resource descriptors before this PR would we get the String value of "GENE" not "gene" yes we could lower case it but its not consistent. This way its always the getDisplayName.